docs_samples/01_python_app_insights_overview.md

```diff
@@ -0,0 +1,224 @@
+  - Select your lab subscription and resource group
+  - Select the **python-appinsights-SUFFIX-mysql** PostgreSQL resource
+  - Under **Settings**, select **Connection security**
+  - Select the **Yes** toggle for the **Allow access to Azure Services**
+  - Select **Add client IP**
+  - Select **Save**
+
+## Configure the environment (Visual Studio Code)
+
+- Open Visual Studio code, select the **Extensions** tab
+- Ensure the following extensions are installed:
+  - ms-python.python
+  - ms-azuretools.vscode-azurefunctions
+
+## Check your Python Version
+
+- In Visual Studio Code, open a Terminal window, run the following command:
+
+```powershell
+python.exe --version
+```
+
+- For Python, make sure you see version 3.9.x.  If you do not, do the following:
+  - Download Python and change your path variables to point to the proper version
+  - Restart Visual Studio Code
+- Open the command palette by selecting **View->Command Pallet**
+- Run the `Python: Select interperter` command
+- Select the `Python 3.9.x` version
+
+## Create a Python environment
+
+- Run the following command to create an environment where you can load all the dependencies:
+
+```powershell
+py -3 -m venv .venv
+.venv\scripts\activate
+```
+
+- When prompted, select **yes**
+- Ensure that you select the new environment in the interpertor selection otherwise the python commands you run later may not map to the proper python version.
+
+## Install OpenCensus
+
+[OpenCensus](https://opencensus.io/) is a set of open source libraries hosted on [GitHub](https://github.com/census-instrumentation) that are used to capture trace and metric data from applications.
+
+Metrics can be things like latency, HTTP request and reponse lengths whereas traces are how a request moves through your application layers and other services. Graphs can be built to show how an application or service is performing based on these data points.
+
+To install OpenCensus:
+
+- Open a terminal window, run the following:
+
+```powershell
+python -m pip install --upgrade pip
+
+python -m pip install opencensus
+```
+
+## Install Azure exporter
+
+OpenCensus needs an exporter to know how to send the log and metric data to a specific backend.  Azure provides an exporter for Azure Monitor and Log Analytics.
+
+To install the Azure monitor exporter package:
+
+- Open a terminal window, run the following:
+
+```powershell
+python -m pip install opencensus-ext-logging
+python -m pip install opencensus-ext-azure
+python -m pip install opencensus-extension-azure-functions
+python -m pip install opencensus-ext-requests
+python -m pip install pyodbc
+python -m pip install psutil
+```
+
+## Configure middleware to track requests
+
+- Browse to the Azure Portal
+- Select the Application Insights **python-appinsights-SUFFIX** resource
+- On the **Overview** page, copy the connection string
+- Open the `.\docs_sampes\.env` file notice that as part of the setup the values have been copied into the environment file
+
```

Comment:
I would include here to verify that the connection strings match from the resource and the settings file.